### DISCUSSION TITLE
Display fields based on the value selected from the dropdown

### DISCUSSION THREAD

**Discussion by Sahith-8055** [rjsf-team/react-jsonschema-form#4036] [Q&A]
Say, I have a dropdown (A). In the dropdown, I would like to have two fields 1 & 2 as options. If the selected value is 1, I would like to showcase two string fields B & C. In a similar manner, if the selected value is 2, I would like to showcase D & E which are string fields. The fields which would appear after the selection of the dropdown option should be hidden initially.

I had tried both the options of oneOf and allOf but to no avail. I tried exploring the Playground but I couldn't get any result close to my use-case. Any suggestions / pointers on how to move ahead with the above said scenario? Thanks in advance!

Below are the references to what I'm trying to ask in the question -

<img width="652" alt="Screenshot 2024-01-10 at 10 10 36 PM" src="https://github.com/rjsf-team/react-jsonschema-form/assets/41897310/e6d30f99-84c4-4bf4-bab5-ff590749b544">
<img width="653" alt="Screenshot 2024-01-10 at 10 11 07 PM" src="https://github.com/rjsf-team/react-jsonschema-form/assets/41897310/738eb6ec-54e4-4071-957c-ab16441fe2e5">

**Comment by codisfy** [comment]
I think what you are looking for is https://rjsf-team.github.io/react-jsonschema-form/docs/json-schema/dependencies#dynamic
